### ISSUE TITLE
Option to redirect to stdout

### ISSUE THREAD

**Issue by Vanuan** [toy/progress#7]
Currently, I have to monkey patch it like this:

``` ruby
module ProgressClassMethods
  def io
    $stdout
  end
end
Progress.send(:extend, ProgressClassMethods)
```

Is there a way to override io method without monkey patching?

**Comment by toy** [comment]
Currently there is no proper way, shorter way would be:
```rb
def Progress.io
  $stdout
end
```

But there can also be a feature to do it like:
```rb
Progress.io = $stdout
```
PR is welcome ;)

**Comment by toy** [comment]
Implemented in b81054e02fb3a5ca1cbba544ed8c07e348c4494c and released in [v3.3.0](https://github.com/toy/progress/releases/tag/v3.3.0)
